lnwallet/channel.go

```diff
@@ -860,6 +877,12 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		theirWitnessScript: theirWitnessScript,
 	}
 
+	// Ensure that we'll restore any custom records which were stored as
+	// extra data on disk.
+	if len(htlc.ExtraData) != 0 {
+		pd.CustomRecords = fn.Some(htlc.ExtraData)
```

Comment:
This for some reason didn't compile for me, needed to add `fn.Some[[]byte](htlc.ExtraData)`...